lib/rubocop/cop/rspec/shared_examples.rb

```diff
@@ -22,7 +22,9 @@ module RSpec
       #
       class SharedExamples < Cop
         def_node_matcher :shared_examples, <<-PATTERN
-          (send nil? {#{(SharedGroups::ALL + Includes::ALL).node_pattern}} $sym ...)
+          (send
+            {(const nil? :RSpec) nil?}
```

Comment:
`{(const nil? :RSpec) nil?}` is being used in a few places now. Is this something we should add to the language.rb file?

Reply:
Definitely. And we generally have to check for both explicit and implicit receiver wherever appilcable

Reply:
I think perhaps the `send_pattern` method in language may generate both explicit/implicit receiver and also provide capture for the arguments?

Reply:
It looks like `SelectorSet#send_pattern` will match something starting with `(send _` – basically setting no restriction on the receiver.

Reply:
Unfortunately we can not use it here because we need the capture, but at least we may also use `_` for consistency and shorter code

Reply:
Ah, I get it now – you were considering using `(SharedGroups::ALL + Includes::ALL).send_pattern` directly as the node pattern. Yeah, that would have been neat.

Ok to merge this PR as is?